docs/src/pages.js

```diff
@@ -161,7 +161,19 @@ const pages = [
     pathname: '/api-docs',
     children: [
       ...findPages[0].children,
-      ...[{ pathname: '/api-docs/data-grid' }, { pathname: '/api-docs/x-grid' }],
+      {
+        pathname: '/api-docs/data-grid',
+        title: 'Data Grid',
+        children: [
+          { pathname: '/api-docs/data-grid', title: 'API Reference' },
+          { pathname: '/api-docs/data-grid/data-grid' },
```

Comment:
Are these pages expected to return 404 at the moment? https://deploy-preview-26477--material-ui.netlify.app/api/data-grid/data-grid/

Reply:
Yes, these pages are available in https://deploy-preview-1529--material-ui-x.netlify.app/api/data-grid/data-grid/.